documentation/modules/deploying/proc-deploy-kafka-connect-new-image-from-base.adoc

```diff
@@ -98,7 +98,7 @@ spec: <1>
     #...
 ----
 <1> link:{BookURLConfiguring}#type-KafkaConnectSpec-reference[The specification for the Kafka Connect cluster^].
-<2> The docker image for the pods.
+<2> The docker image for generating Kafka Connect pods.
```

Comment:
Again, I would avoid the use of the `generating` word. I think the original statement was better.